CONTRIBUTING.md

```diff
@@ -15,13 +18,14 @@ Please see https://git.k8s.io/community/CLA.md for more info.
 - Access to a Kubernetes v1.11.3+ cluster.
 
 ## Contributing steps
+
 1. Submit an issue describing your proposed change to the repo in question.
 1. The [repo owners](OWNERS) will respond to your issue promptly.
 1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
 1. Fork the desired repo, develop and test your code changes.
 1. Submit a pull request.
 
-## How to build kubebuilder locally
+## How to build kubebuilder locally?
```

Comment:
I would say that without the question mark looks better. It is a set of instruction, not a question.

Reply:
Makes sense. Thank you 👍